site/content/docs/user/quick-start.md

```diff
@@ -242,6 +242,49 @@ nodes:
 - role: worker
 ```
 
+### Enable Feature Gates in Your Cluster
```

Comment:
should this be in another page? the "quick start" seems to be covering a lot now :thinking:

Reply:
indeed, but I think that we should first thing how do we want to reorganize the docs, seems that they are starting to grow and I think that adding an advanced section with this stuff is not a good solution because we are going to have the same problem after some time.